example/layers.js

```diff
@@ -264,6 +264,9 @@ var embedControl = new L.control.embed({
 });
 embedControl.addTo(map);
 
+var modeControl = new L.control.minimalMode(leafletControl);
```

Comment:
So many features :P 
We need to make readme.md readble!!!

Reply:
Yes. I will be doing that this weel. :)